addons/website/views/website_views.xml

```diff
@@ -387,6 +387,38 @@ action = {
             <field name="sequence">0</field>
         </record>
 
+        <record id="theme_view_form_preview" model="ir.ui.view">
+            <field name="name">website.form</field>
+            <field name="model">ir.module.module</field>
+            <field name="mode">primary</field>
+            <field name="arch" type="xml">
+                <form create="false" edit="false" delete="0" js_class="theme_preview_form">
+                  <div class="preview_frame h-100">
+                        <field name='url' widget='iframe'/>
+                        <img alt='' class='img_mobile' style='display:none' src="/website/static/src/img/phone.png"/>
+                  </div>
+
+                </form>
+            </field>
+        </record>
+
+        <record id="theme_install_kanban_action" model="ir.actions.act_window">
+            <field name="name">Choose a theme for your website</field>
+            <field name="res_model">ir.module.module</field>
+            <field name="view_mode">kanban,form</field>
+            <field name="view_id" ref="website.theme_view_kanban" />
+            <field name="target">fullscreen</field>
+            <field name="view_ids"
+                   eval="[(5, 0, 0),
+                          (0, 0, {'view_mode': 'kanban', 'view_id': ref('website.theme_view_kanban')}),
+                          (0, 0, {'view_mode': 'form', 'view_id': ref('website.theme_view_form_preview')})]"/>
```

Comment:
Is this really how we must do that in Odoo... damn... :smile: I don't understand why it is not
```python
[(6, 0, [ref(A), ref(B)])]
```

Reply:
not really

`(6, 0, [ref(A), ref(B)]) `== `(5, 0, 0), (4, ref(A), 0)), (4, ref(B), 0))`

But in this case, element is not yet created, so we create at fly with the (0,0, {values})

```
X = create(values)
(4, ref(X), 0)) 
```
=== 
`(0,0, {values})`